config/routes.rb

```diff
@@ -8,6 +8,6 @@
   resources :admin, only: [:index]
   namespace :admin do
     resources :merchants, only: [:index]
-    resources :invoices, only: [:index]
+    resources :invoices, only: [:index, :show]
```

Comment:
nice add to the resources! this will give us the admin_invoice_path, ie, "/admin/invoices/:id"